SocialCareCaseViewerApi/V1/Controllers/CaseStatusTypeFieldsController.cs

```diff
@@ -0,0 +1,44 @@
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using SocialCareCaseViewerApi.V1.Boundary.Requests;
+using SocialCareCaseViewerApi.V1.Boundary.Response;
+using SocialCareCaseViewerApi.V1.UseCase;
+using SocialCareCaseViewerApi.V1.UseCase.Interfaces;
+
+namespace SocialCareCaseViewerApi.V1.Controllers
+{
+    [ApiController]
+    [Route("api/v1")]
+    [Produces("application/json")]
+    [ApiVersion("1.0")]
+    public class CaseStatusTypeFieldsController : BaseController
+    {
+        private readonly IGetCaseStatusFieldsUseCase _getCaseStatusFieldsUseCase;
+
+        public CaseStatusTypeFieldsController(IGetCaseStatusFieldsUseCase getCaseStatusFieldsUseCase)
+        {
+            _getCaseStatusFieldsUseCase = getCaseStatusFieldsUseCase;
+        }
+
+        /// <summary>
+        /// Get a list of fields and their options for a case status type
+        /// </summary>
+        /// <response code="200">Found case status type, returns the fields for that type</response>
+        /// <response code="404">Case status type not found, or no fields exist</response>
+        [ProducesResponseType(typeof(GetCaseStatusFieldsResponse), StatusCodes.Status200OK)]
+        [HttpGet]
+        [Route("/case-status/form-options/{type:alpha}")]
+        public IActionResult GetCaseStatusTypeFields(string type)
```

Comment:
Incoming string should probably have validation. If the string is not provided, the API will simply return 404 in this instance. Adding validation would provide correct status code